### PR TITLE
Digest

### PR DESCRIPTION
**Pull request by Sarahtonein**
Removed PYTZ import from bot.py
Added aggregate func to notion.py to grab items from last 7 days with the tag "digest"
Runs 1x a week

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 COLLECTIVE_DB = "07752fd5ba8e44c7b8e48bfee50f0545?"
 DATE_FMT = "%d %m %Y %H:%M"
 
-NOTION_NOTIFICATION_CHANNEL = 1126132159158173726
```
Comment (Balugrizzly) [inline constants.py]: Why did you change from the notion-hookz discord chan to the testing chan?

This probably should be reverted to the notion-hookz chan.

**Review by Balugrizzly — CHANGES_REQUESTED** [review]
The only major thing is the NOTION_NOTIFICATION_CHANNEL that has been changed without reason.

Discussion points of minor importance, i.e error handling and logging have remained unchanged.

File: constants.py (hunk 1 above)
Comment (Sarahtonein) [reply constants.py]: Changed to testing chan while I was experimenting with the aggregate changes. 

Apologies on the error handling / logging, I made changes but when I reset a prior commit I did not add them back.

Comment (Balugrizzly) [reply constants.py]: @Sarahtonein Can you fix it?
You can add the error handling and logging later in a subsequent commit/branch/PR if you want

Comment (Sarahtonein) [reply constants.py]: @Balugrizzly ,

I've changed the channel back to Notion Hooks. Will do the error handling in a subsequent commit
